src/main/app/jurisdiction/connections.ts

```diff
@@ -63,26 +63,32 @@ const onlyApplicant1Domiciled = data => {
   return isDomiciled('applicant1', data) && !isDomiciled('applicant2', data);
 };
 
-export const allowedToAnswerResidualJurisdiction = (data: Partial<CaseWithId>): boolean => {
+export const allowedToAnswerResidualJurisdiction = (
+  data: Partial<CaseWithId>,
+  connections: JurisdictionConnections[] | undefined
```

Comment:
would this ever be undefined?
because were setting connections to an empty array each time and passing that.